### PR TITLE
docs: New multicluster documentation

### LINKED ISSUE #355: doc: Multicluster documentation
Write some documentation about the multi cluster setup using Razee.

### PR DESCRIPTION
**Pull request by froesef**
Fixes #355

### CODE REVIEW

File: site/mkdocs.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ nav:
 - Get Started:
   - get-started/quickstart.md
   - samples/notebook.md
+  - get-started/multicluster.md
```
Comment (roee88) [inline site/mkdocs.yml:15]: I don't think this is the right place. Suggestions for the sections would be appropriated.

I originally planned for quickstart to have just default install and notebook sample, and add another section for Install for custom installation instructions.

File: site/docs/get-started/multicluster.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+recommended. 
+It's possible to define a multicluster group name that groups clusters that are used in a mesh instance. This will restrict the clusters that 
+are usable in the mesh instance to the ones that are registered in the specified Razee group. This is especially helpful if Razee is also 
+used for different purposes than Mesh for data or multiple mesh instances should be used under the same Razee installation.
+
+In general there is a need for the following Razee components to be installed:
+
+* Razee watch keeper (installed on all clusters)
+* Razee cluster subscription manager (installed on all clusters)
+* RazeeDash API (installed on controller cluster/as cloud service)
```
Comment (roee88) [inline site/docs/get-started/multicluster.md:21]: coordinator cluster

**Review by roee88 — COMMENTED** [review]
The content itself is great, but for formatting and style I would have preferred a step by step approach instead. We can iterate on it in the future though. Let's decide where this page fits.

File: site/mkdocs.yml (hunk 1 above)
Comment (froesef) [reply site/mkdocs.yml]: How about having it in the tasks section?

**Review by roee88 — CHANGES_REQUESTED** [review]
Can you please:
1. Rebase
2. Remove the changes to site/docs/reference (we need a a separate PR to add them to gitignore and remove from the git index)
3. Add a section for installing M4D on a remote cluster (`--coordinator.enabled=false`).

**Review by roee88 — APPROVED** [review]

**Pull request merged**